tools/topology/sof-smart-amplifier.m4

```diff
@@ -41,6 +41,8 @@ ifdef(`SMART_SSP_NAME',`',`errprint(note: Need to define SSP BE dai_link name fo
 )')
 # define(`SMART_SSP_QUIRK', 0) define SSP quirk
 ifdef(`SMART_SSP_QUIRK',`',`define(`SMART_SSP_QUIRK', 0)')
+# define(`SSP_MCLK', ) define SSP mclk if not done yet
+ifdef(`SSP_MCLK',`',`define(`SSP_MCLK', 38400000)')
```

Comment:
we don't need to output 38.4 for the codec to work, 19.2 is enough and is safer (will have fewer issues on the line).

Reply:
Good point, let me change the default to 19.2, but keep it configurable from the upper level .m4.